docs/events.md

```diff
@@ -55,27 +55,42 @@ utilizes a listener (**ExceptionLogGroupCreatedListener)** to listen for this
 event and sends out a notification (**ExceptionLogGroupNotification)** to all
 users, notifying them of the creation.
 
-### ExceptionLogGroupUpdatedEvent
+## ExceptionLogGroupUpdatedEvent
 
 This event is triggered when an existing exception log group is modified. This
 is useful when multiple repeated exceptions are received in a single batch.
 MoonGuard utilizes a listener (**ExceptionLogGroupUpdatedListener)** to listen
 for this event and sends out a notification (**ExceptionLogGroupNotification)**
 to all users, notifying them of the update.
 
+## SystemMetricAlertEvent
+
+This event is triggered when the CPU load, memory, or disk space exceeds the
+limit set in the site configuration, causing an alert. MoonGuard utilizes a listener
+(**SystemMetricAlertListener**) to liste for this event and sends out notification
+(**SystemMetricNotification**) to all users, notifying them of the update.
+
 If you need to modify the behavior of the Listener, they can be replaced by your
 own listeners from the MoonGuard configuration file:
```

Comment:
Replace with:

"In case you want to customize the behavior of this event listeners, they can be replaced by your own listeners in the MoonGuard configuration file:"